### PR TITLE
[ci] [doc] More detailed documentation for overlay building

### PR DESCRIPTION
**Pull request by olaure01**
Proposal of additional instructions for using [create_overlays.sh](https://github.com/coq/coq/blob/master/dev/tools/create_overlays.sh).
I am not sure this is the exact intended protocol but it works well for me.

**Kind:** documentation / infrastructure.

### CODE REVIEW

File: dev/ci/README-developers.md [hunk 1]
```diff
@@ -73,16 +73,31 @@ Moreover your PR must absolutely update the [`CHANGES.md`](../../CHANGES.md) fil
...
+
+    replacing `ejgallego` by your GitHub nickname, `9873` by the actual PR
+number, and selecting the XXXi hosted on GitHub. The script will:
+
+    + checkout the contributions and prepare the branch/remote so you can
+      just commit the fixes and push,
+    + add the corresponding overlay file in `dev/ci/user-overlays`;
+
+- go to `_build_ci/XXXi` to prepare your overlay
+(you can test your modifications by using `../../make ci-XXXi`)
```
Comment (SkySkimmer) [inline dev/ci/README-developers.md:97]: I would be shocked if that works, maybe you mean
```suggestion
(you can test your modifications by using `make -C ../.. ci-XXXi`)
```

Comment (olaure01) [reply dev/ci/README-developers.md:97]: Absolutely. What I do in practice is `cd ../..` and then `make ci-XXXi`. I was looking for a simpler way to describe it, but I failed... Thanks for your input.

**Comment by Zimmi48** [comment]
LGTM

**Comment by SkySkimmer** [comment]
@Zimmi48 you want to merge?

**Comment by olaure01** [comment]
I am going to merge commits right now.

**Review by Zimmi48 — APPROVED** [review]
Sure, let's not delay this any longer.

**Pull request merged**